src/main/java/com/easypost/model/TimeInTransit.java

```diff
@@ -7,146 +7,148 @@
+        this.percentile85 = percentile85;
     }
 
     /**
      * Get the 90th percentile of this TimeInTransit.
      *
      * @return the 90th percentile of this TimeInTransit
      */
-    public Integer getPercentile_90() {
-        return percentile_90;
+    public Integer getPercentile90() {
+        return percentile90;
     }
 
     /**
      * Set the 90th percentile of this TimeInTransit.
      *
-     * @param percentile_90 the 90th percentile of this TimeInTransit
+     * @param percentile90 the 90th percentile of this TimeInTransit
      */
-    public void setPercentile_90(final Integer percentile_90) {
-        this.percentile_90 = percentile_90;
+    public void setPercentile90(final Integer percentile90) {
+        this.percentile90 = percentile90;
     }
 
     /**
      * Get the 95th percentile of this TimeInTransit.
      *
      * @return the 95th percentile of this TimeInTransit
      */
-    public Integer getPercentile_95() {
-        return percentile_95;
+    public Integer getPercentile95() {
+        return percentile95;
     }
 
     /**
      * Set the 95th percentile of this TimeInTransit.
      *
-     * @param percentile_95 the 95th percentile of this TimeInTransit
+     * @param percentile95 the 95th percentile of this TimeInTransit
      */
-    public void setPercentile_95(final Integer percentile_95) {
-        this.percentile_95 = percentile_95;
+    public void setPercentile95(final Integer percentile95) {
+        this.percentile95 = percentile95;
     }
 
     /**
      * Get the 97th percentile of this TimeInTransit.
      *
      * @return the 97th percentile of this TimeInTransit
      */
-    public Integer getPercentile_97() {
-        return percentile_97;
+    public Integer getPercentile97() {
+        return percentile97;
     }
 
     /**
      * Set the 97th percentile of this TimeInTransit.
      *
-     * @param percentile_97 the 97th percentile of this TimeInTransit
+     * @param percentile97 the 97th percentile of this TimeInTransit
      */
-    public void setPercentile_97(final Integer percentile_97) {
-        this.percentile_97 = percentile_97;
+    public void setPercentile97(final Integer percentile97) {
+        this.percentile97 = percentile97;
     }
 
     /**
      * Get the 99th percentile of this TimeInTransit.
      *
      * @return the 99th percentile of this TimeInTransit
      */
-    public Integer getPercentile_99() {
-        return percentile_99;
+    public Integer getPercentile99() {
+        return percentile99;
     }
 
     /**
      * Set the 99th percentile of this TimeInTransit.
      *
-     * @param percentile_99 the 99th percentile of this TimeInTransit
+     * @param percentile99 the 99th percentile of this TimeInTransit
      */
-    public void setPercentile_99(final Integer percentile_99) {
-        this.percentile_99 = percentile_99;
+    public void setPercentile99(final Integer percentile99) {
+        this.percentile99 = percentile99;
     }
 }
```

Comment:
I see stuff like this and I wonder why we can't just use a struct-like class with public members and no accessors.